corehq/apps/app_manager/tests/data/suite/advanced_module_parent_null_relationship.xml

```diff
@@ -0,0 +1,15 @@
+<partial>
+  <entry>
+    <form>http://id_m1-f0</form>
+    <command id="m1-f0">
+      <text>
+        <locale id="forms.m1f0"/>
+      </text>
+    </command>
+    <instance id="casedb" src="jr://instance/casedb"/>
+    <session>
+      <datum id="parent_id" nodeset="instance('casedb')/casedb/case[@case_type='parent'][@status='open']" value="./@case_id" detail-select="m0_case_short"/>
+      <datum id="case_id" nodeset="instance('casedb')/casedb/case[@case_type='child'][@status='open']" value="./@case_id" detail-select="m1_case_short" detail-confirm="m1_case_long"/>
```

Comment:
@orangejenny Just curious here, this datum config means that there is no relation between these two menus. So, the second menu will pull all cases of the lastly selected module. So the workflow would be to select two cases of no relationship. Is that what's intended with this change?

Reply:
Yeah, exactly. The use case for this is deduping cases, so people will be picking two modules with the same case type (come to think of it, we could limit the dropdown to only modules of the same case type...might be a good thing to chat with Nick about). They'll use the first datum to select any case. Then the second datum will be the same case type, and it'll be a module that uses the new [search-first case claim](https://github.com/dimagi/formplayer/pull/783) and also uses the new [default value for case claim prompts](https://github.com/dimagi/commcare-core/pull/959) functionality. Those default values will have expressions that reference the first case. So the ultimate UX is that you search for a patient, select your patient, and then you're taken to a case search screen that's already populated with the name, phone number, etc. from that first patient, so you just click "Search" and that then brings up a list of likely duplicate patients.

Reply:
Makes sense, thanks Jenny! Except that we may have to limit the dropdown to modules of the same case-type only, is this the suite change that we intended?

Reply:
yep!